osu.Game/Screens/Menu/IntroCircles.cs

```diff
@@ -61,8 +61,8 @@ private void load(OsuConfigManager config, BeatmapManager beatmaps, Framework.Ga
                 welcome = samples.Get(@"welcome");
         }
 
-        private const double delay_step_one = 2300;
-        private const double delay_step_two = 600;
+        private const int delay_step_one = 2300;
```

Comment:
same here, why is it `int` when it's usage requires type `double` (e.g `IntroCircles:L89`)